plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -114,18 +112,26 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
         }
     }
 
-    fun handleProgressStateChanged(newState: TransformationStatus, transformationPlan: TransformationPlan?, jdkVersion: JavaSdkVersion) {
+    fun handleProgressStateChanged(newState: TransformationStatus, plan: TransformationPlan?, jdk: JavaSdkVersion, transformType: CodeTransformType) {
         val currentState = statusTreePanel.getCurrentElements()
         val loadingPanelText: String
         // show the details panel when there are progress updates
         // otherwise it would show an empty panel
         val backendProgressStepsAvailable = (
-            transformationPlan != null &&
-                transformationPlan.hasTransformationSteps() &&
-                haveProgressUpdates(transformationPlan)
+            plan != null &&
+                plan.hasTransformationSteps() &&
+                haveProgressUpdates(plan)
             )
 
         fun maybeAdd(stepId: ProgressStepId, string: String) {
```

Comment:
another naming nit: could this function be named a bit clearer to indicate what it's doing?

Reply:
Yes, good point, will fix